### PR TITLE
22.2 rotated text

### PR DESCRIPTION
**Pull request by geumjaLee**

### CODE REVIEW

File: interface-elements-for-desktop/articles/rich-text-editor/tables/rotate-table-cell-content.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Rotate Table Cell Content
+author: Polina Fedorova
+---
+# Rotate Table Cell Content
+
+The Rich Text Editor allows you to change the rotation type of the table cell content.
```
Comment (arminal) [inline interface-elements-for-desktop/articles/rich-text-editor/tables/rotate-table-cell-content.md:7]: The Rich Text Editor allows you to change the table cell content's rotation type.

File: interface-elements-for-desktop/articles/rich-text-editor/tables/rotate-table-cell-content.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+1. Place the cursor within the table cell which content you wish to rotate.
```
Comment (arminal) [inline interface-elements-for-desktop/articles/rich-text-editor/tables/rotate-table-cell-content.md:9]: 1. Place the cursor on the table cell whose content you wish to rotate.

File: interface-elements-for-desktop/images/eud-tables-text-rotation.png
Comment (arminal) [inline interface-elements-for-desktop/images/eud-tables-text-rotation.png:1]: Under 5 Conclusion, it says "Ounce will overcome..." What is Ounce?

Comment (geumjaLee) [reply interface-elements-for-desktop/images/eud-tables-text-rotation.png:1]: Ounce is a term used throughout this text. This is an artificially generated pseudo-scientific article

**Pull request merged**